device/arista/x86_64-arista_7060_cx32s/Arista-7060CX-32S-Q24C8/buffers_defaults_t0.j2

```diff
@@ -10,14 +10,14 @@
         {%- if PORT_ALL.append("Ethernet%d" % (port_idx * 4)) %}{%- endif %}
```

Comment:
The port mappings for this one look wrong. All of them have 4 lanes as per the port_config.ini

Reply:
Good catch. That file was wrong from day 1 :-)